e2e/rook-test-framework/clients/rook_client.go

```diff
@@ -0,0 +1,102 @@
+package clients
+
+import (
+	"errors"
+	"github.com/rook/rook/e2e/rook-test-framework/contracts"
+	"github.com/rook/rook/e2e/rook-test-framework/enums"
+	"github.com/rook/rook/e2e/rook-test-framework/transport"
+)
+
+var (
+	STATUS_CMD  = []string{"rook", "status"}
+	VERSION_CMD = []string{"rook", "version"}
+	NODE_CMD    = []string{"rook", "node"}
```

Comment:
the command `rook node` by itself isn't going to do anything but print out the help.  you'd need another command such as `ls`.

note i'm still questioning if going through the `rook` CLI is the right approach vs. going through the client library.